tests/migrations/test_base.py

```diff
@@ -130,3 +132,154 @@ def temporary_migration_module(self, app_label='migrations', module=None):
                 new_module = os.path.basename(target_dir) + '.migrations'
                 with self.settings(MIGRATION_MODULES={app_label: new_module}):
                     yield target_migrations_dir
+
+
+class OperationTestBase(MigrationTestBase):
```

Comment:
As explained in [d68bafc](https://github.com/django/django/pull/10901/commits/d68bafca1f8b2e34dcc80ba0beaa02103fed0235), I moved this class here to make it more readily importable elsewhere (namely, in tests/postgres_tests/test_operations.py).